tests/sentry/tasks/test_groupowner.py

```diff
@@ -0,0 +1,264 @@
+from sentry.tasks.groupowner import process_suspect_commits, PREFERRED_GROUP_OWNER_AGE
+from sentry.testutils import TestCase
+from sentry.testutils.helpers.datetime import iso_format, before_now
+from sentry.models import Repository, GroupOwner
+from sentry.utils.committers import get_serialized_event_file_committers
+
+
+class TestGroupOwners(TestCase):
+    def setUp(self):
+        self.project = self.create_project()
+        self.repo = Repository.objects.create(
+            organization_id=self.organization.id, name=self.organization.id
+        )
+        self.release = self.create_release(project=self.project, version="v1337")
+        self.group = self.create_group(
+            project=self.project, message="Kaboom!", first_release=self.release
+        )
+
+        self.event_1 = self.store_event(
+            data={
+                "message": "Kaboom!",
+                "platform": "python",
+                "timestamp": iso_format(before_now(seconds=10)),
+                "stacktrace": {
+                    "frames": [
+                        {
+                            "function": "handle_set_commits",
+                            "abs_path": "/usr/src/sentry/src/sentry/tasks.py",
+                            "module": "sentry.tasks",
+                            "in_app": True,
+                            "lineno": 30,
+                            "filename": "sentry/tasks.py",
+                        },
+                        {
+                            "function": "set_commits",
+                            "abs_path": "/usr/src/sentry/src/sentry/models/release.py",
+                            "module": "sentry.models.release",
+                            "in_app": True,
+                            "lineno": 39,
+                            "filename": "sentry/models/release.py",
+                        },
+                    ]
+                },
+                "tags": {"sentry:release": self.release.version},
+                "fingerprint": ["put-me-in-the-control-group"],
+            },
+            project_id=self.project.id,
+        )
+
+    def set_release_commits(self, author_email):
+        self.release.set_commits(
+            [
+                {
+                    "id": "a" * 40,
+                    "repository": self.repo.name,
+                    "author_email": author_email,
+                    "author_name": "Bob",
+                    "message": "i fixed a bug",
+                    "patch_set": [{"path": "src/sentry/models/release.py", "type": "M"}],
+                }
+            ]
+        )
+
+    def test_simple(self):
+        self.set_release_commits(self.user.email)
+
+        result = get_serialized_event_file_committers(self.project, self.event_1)
+
+        assert len(result) == 1
+        assert "commits" in result[0]
+        assert len(result[0]["commits"]) == 1
+        assert result[0]["commits"][0]["id"] == "a" * 40
```

Comment:
Are these tests here to help test `get_serialized_event_file_committers()`? They don't seem connected to what this test is actually aiming to do at first glance.

Reply:
Yeah. I just removed them.